### PR TITLE
[BPK-4264] Migrate CI to GitHub Actions

### PR DESCRIPTION
**Pull request by shaundon**

### CODE REVIEW

**Review by georgegillams — APPROVED** [review]

**Comment by shaundon** [comment]
@georgegillams thanks for the approval but this is WIP and CI failed 😉

**Review by georgegillams — CHANGES_REQUESTED** [review]
this is WIP and CI failed

**Comment by georgegillams** [comment]
I fixed my review @shaundon 😂

**Comment by shaundon** [comment]
Hahaha I love it 😆

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Backpack Docs CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  repository_dispatch:
+    types: [rebuild]
```
Comment (shaundon) [inline .github/workflows/ci.yml:9]: This is what we'll use to trigger this workflow remotely via a curl from other workflows

Comment (georgegillams) [reply .github/workflows/ci.yml:9]: Oh nice!

**Review by georgegillams — APPROVED** [review]
LGTM

**Pull request merged**